app/containers/SwimlaneContainer.js

```diff
@@ -111,6 +114,9 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
     },
     onTaskUpdated: (task, updateAsana) => {
       dispatchProps.onTaskUpdated(task, id, currentProjectId, updateAsana);
+    },
+    onSwimlaneToggle: (id) => {
```

Comment:
The `mergeProps` function is only required when you need to access the container's state props in the`mapDispatchToProps` as to my knowledge it isn't possible in `mapDispatchToProps`. As you don't need access the state props due to the swimlane id being passed through from the component the duplication isn't necessary as the function you declared above will be used. If you look at the end of `mergeProps` you'll see them all being merged together.